highway_env/envs/parking_env.py

```diff
@@ -131,60 +141,152 @@ def _create_road(self, spots: int = 14) -> None:
 
         :param spots: number of spots in the parking
         """
+        spots = self.config["spots"]
         net = RoadNetwork()
         width = 4.0
         lt = (LineType.CONTINUOUS, LineType.CONTINUOUS)
         x_offset = 0
-        y_offset = 10
+        y_offset = self.config["y_offset"]
         length = 8
-        for k in range(spots):
-            x = (k + 1 - spots // 2) * (width + x_offset) - width / 2
-            net.add_lane("a", "b", StraightLane([x, y_offset], [x, y_offset+length], width=width, line_types=lt))
-            net.add_lane("b", "c", StraightLane([x, -y_offset], [x, -y_offset-length], width=width, line_types=lt))
+
+        row_ys = []
```

Comment:
I think this could be simplified, why is this list needed? is the y coordinate not just a linear function of the row index? maybe make this a lambda function.